tests/lib/prepare.sh

```diff
@@ -261,7 +261,13 @@ EOF
     # the service setting may have changed in the service so we need
     # to ensure snapd is reloaded
     systemctl daemon-reload
+
+    # Leave some time for snapd to finish processing hooks
+    sleep 5
```

Comment:
Really awesome find!! I just have a question, will 5 seconds *always* be enough? Or is there something else we could poll for? (Just thinking in terms of robustness)

Reply:
Testing in https://github.com/snapcore/core-base/pull/108

Reply:
It worked. Done.